app/models/user.rb

```diff
@@ -313,6 +313,33 @@ def compressed_export
     ActiveSupport::Gzip.compress Diaspora::Exporter.new(self).execute
   end
 
+  ######### Photos export ##################
+  mount_uploader :exported_photos_file, ExportedPhotos
+
+  def queue_export_photos
+    update exporting_photos: true
+    Workers::ExportPhotos.perform_async(id)
+  end
+
+  def perform_export_photos!
+    export = Tempfile.new([username, '_photos.zip'])
+    
+    Zip::OutputStream.open(export) { |zos| }
+
+    Zip::ZipFile.open(export.path, Zip::File::CREATE) do |zipfile|
+      photos.each do |photo|
+          photo_path = "#{Rails.root}/public/uploads/images/#{photo.remote_photo_name}"
```

Comment:
Oh! You're talking about line 331. I thought you were talking about line 317.

I assume that files are stored in same machine as web server. :-(